evaluate_server.py

```diff
@@ -0,0 +1,24 @@
+"""
+Server to handle processes of evaluating.
+"""
+
+import pickle
+from flask import Flask
+from flask import request
+from thousandaire.evaluator import Evaluator
+
+APP = Flask(__name__)
+
+@APP.route('/', methods=['GET', 'POST'])
```

Comment:
Why do we treat GET and POST equivalently here?
I think using one of them is enough.